src/Core/src/Handlers/Page/PageHandler.Android.cs

```diff
@@ -1,14 +1,20 @@
-﻿using Android.Views;
-using Microsoft.Maui.Graphics;
-
-namespace Microsoft.Maui.Handlers
+﻿namespace Microsoft.Maui.Handlers
 {
 	public partial class PageHandler : ContentViewHandler
 	{
-		//Graphics.Color? DefaultBackgroundColor;
-
 		public static void MapTitle(IPageHandler handler, IContentView page)
 		{
 		}
+
+		public static void MapBackgroundImageSource(IPageHandler handler, IContentView page)
```

Comment:
Can we move all of this up into ViewHandler? 

The extension methods all seem to work against UIView/Android.View/Panel

WinUI is the only one that has kind of a specific type but could probably just use pattern matching to look through the types that accept a background. 

This way anyone could annotate any view with `IViewBackgroundImagePart` and then in controls we will only implement `IViewBackgroundImagePart` at the page level